services/circleci/circleci.service.js

```diff
@@ -2,70 +2,48 @@
 
 const Joi = require('joi')
 const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
-const { BaseJsonService } = require('..')
+const { BaseSvgScrapingService, redirector } = require('..')
 
-const circleSchema = Joi.array()
-  .items(Joi.object({ status: isBuildStatus }))
-  .min(1)
-  .max(1)
-  .required()
+const circleSchema = Joi.object({ message: isBuildStatus }).required()
+const queryParamSchema = Joi.object({ token: Joi.string() }).required()
 
 const documentation = `
   <p>
-    Please note that <code>status</code> tokens will not work. Instead, you should generate an <code>all</code> scoped token.
+    You may specify an optional token to get the status for a private repository.
     <br />
-    For the sake of security, please use <b>Project Tokens</b> and never <b>Personal Tokens</b> as they grant full read write permissions to your projects.
+    If you need to use a token, please use a <b>Project Token</b> and only assign your token the 'Status' permission. Never use a <b>Personal Token</b> as they grant full read write permissions to your projects.
     <br />
     For more information about managing Circle CI tokens, please read this <a target="_blank" href="https://circleci.com/docs/2.0/managing-api-tokens">article</a>.
   </p>
   `
 
-module.exports = class CircleCi extends BaseJsonService {
+const vcsTypeMap = { github: 'gh', bitbucket: 'bb' }
+
+class CircleCi extends BaseSvgScrapingService {
   static get category() {
     return 'build'
   }
 
   static get route() {
     return {
-      base: 'circleci',
-      format:
-        '(?:token/(\\w+)/)?project/(?:(github|bitbucket)/)?([^/]+/[^/]+)(?:/(.*))?',
-      capture: ['token', 'vcsType', 'userRepo', 'branch'],
+      base: 'circleci/build',
+      pattern: ':vcsType(gh|bb)/:user/:repo/:branch*',
```

Comment:
I think the vcs options should be the more user friendly terms no?
```suggestion
      pattern: ':vcsType(github|bitbucket)/:user/:repo/:branch*',
```